app/models/IvdSubmission.scala

```diff
@@ -85,6 +85,29 @@ object IvdSubmission extends FixedConfig {
       Json.obj()
     }
 
+    val defermentDetails = if (data.defermentAccountNumber.isDefined) {
```

Comment:
Rewrite to remove the use of `.get` on options. Also, deferment details should only be built if the user has said they want to pay by deferment (bool currently missing from model) rather than if the data exists. With this implementation, if a user:

1. selected pay by deferment
2. entered the deferment details
3. clicked continue
4. Clicked back all the way back to the pay by deferment details page
5. selected not to pay by deferment
6. submitted

The deferment details would be included in the submission.